CONTRIBUTING.rst

```diff
@@ -100,16 +90,74 @@ To get tox, just pip install them into your virtualenv.
 
 7. Submit a pull request through the GitHub website.
 
-***********************
-Pull Request Guidelines
-***********************
+Development tips
+----------------
+
+This project allows you to use `pre-commit <https://pre-commit.com/>`_ to ensure an easy compliance
+to the project code styles.
 
-Before you submit a pull request, check that it meets these guidelines:
+If you want to use it, install it globally (for example with ``pip3 install --user precommit``,
+but check `installation instruction <https://pre-commit.com/#install>`_.
+When first cloning the project ensure you install the git hooks by running ``pre-commit install``.
 
-1. The pull request should include tests.
-2. If the pull request adds functionality, the docs should be updated. Put
-   your new functionality into a function with a docstring, and add the
-   feature to the list in README.rst.
-3. The pull request should work for Python 2.7, 3.4, 3.5 and 3.6. Check
-   https://travis-ci.org/nephila/giturlparse/pull_requests
-   and make sure that the tests pass for all supported Python versions.
+From now on every commit will be checked against our code style.
+
+Check also the available tox environments with ``tox -l``: the ones not marked with a python version number are tools
+to help you work on the project buy checking / formatting code style, running docs etc.
+
+Testing tips
+------------
+You can test your project using any specific combination of python, django and django cms.
+
+For example ``tox -epy3.7-django30-cms37`` runs the tests on python 3.7, Django 3.0 and django CMS 3.7.
```

Comment:
adapt environment name to an existing one

Reply:
@yakky Fixed